bixlight/src/main/java/com/github/shingyx/bixlight/BixlightService.java

```diff
@@ -39,12 +42,18 @@ public void onAccessibilityEvent(AccessibilityEvent event) {
                 "onAccessibilityEvent: [type] %s [time] %s [activeWindowPackage] %s",
                 AccessibilityEvent.eventTypeToString(event.getEventType()), event.getEventTime(), activeWindowPackage));
 
-        if (!BIXBY_PACKAGE.equals(activeWindowPackage)) {
+        long currentMillis = Calendar.getInstance().getTimeInMillis();
```

Comment:
I prefer `System.currentTimeMillis();` to avoid making a Calendar instance. More of a micro-optimisation.